app/routes/product-details.$productSlug/route.tsx

```diff
@@ -22,6 +22,16 @@ export const loader = ({ params, request }: LoaderFunctionArgs) => {
     return getProductDetailsRouteData(params.productSlug, request.url);
 };
 
+export const getStaticRoutes = async () => {
```

Comment:
For a user it's going to be difficult to understand what this function is, since it's not a Remix feature. Do we have a type that we could use here to make it more clear what it is for and to help find documentation?

Something like: `export const getStaticRoutes: CoduxStaticRoutes = async`, where hovering on `CoduxStaticRoutes` provides a link to the documentation. Even without a link knowing that it's related to Codux would help already.

Reply:
Thinking about where this type would be imported from... `@wixc3/define-remix-app`?

It's a dev dependency, but since we'd be only importing a type, it should be fine.

Reply:
Nice one! On it.